modules/markup/sanitizer.go

```diff
@@ -65,8 +65,8 @@ func ReplaceSanitizer() {
 	// Allow classes for emojis
 	sanitizer.policy.AllowAttrs("class").Matching(regexp.MustCompile(`emoji`)).OnElements("img")
 
-	// Allow icons, checkboxes and emojis on span
-	sanitizer.policy.AllowAttrs("class").Matching(regexp.MustCompile(`^((icon(\s+[\p{L}\p{N}_-]+)+)|(ui checkbox)|(ui checked checkbox)|(emoji))$`)).OnElements("span")
+	// Allow icons, checkboxes, emojis, and chroma syntax on span
+	sanitizer.policy.AllowAttrs("class").Matching(regexp.MustCompile(`^((icon(\s+[\p{L}\p{N}_-]+)+)|(ui checkbox)|(ui checked checkbox)|(emoji))$|^((c|g|i|k|m|n|o|s|u|v|w)[a-z0-9]{0,2})$`)).OnElements("span")
```

Comment:
What this regexp is meant to do?

Reply:
It is for the classes used for highlighting (see chroma.less in this PR for the classes) All Markdown is run through sanitizer so we need to allow these for the Chroma markdown plugin generated code to work.

So if in markdown you wrote
```

```go
t.Fatal(err)
``
```
It would highlight it by producing this:

```<span class="nx">t</span>
<span class="p">.</span>
<span class="nf">Fatal</span>
<span class="p">(</span>
<span class="nx">err</span>
<span class="p">)</span>
```


Similar to what it is like on Github (they used to use Pigments which this is based on but I think use something else now). 

I know these regex are getting a bit out of hand when we keep adding new things but wasn't sure if there was a better option. They don't seem to stack either from what I can tell so everything needs to be inside one big one ; (

Reply:
Maybe we could just use such:
```suggestion
	sanitizer.policy.AllowAttrs("class").Matching(regexp.MustCompile(`^((icon(\s+[\p{L}\p{N}_-]+)+)|(ui checkbox)|(ui checked checkbox)|(emoji))$|^([a-z][a-z0-9]{0,2})$`)).OnElements("span")
```

Reply:
sure, done